### ISSUE TITLE
Images change when using the command Copy-ExcelWorkSheet

### ISSUE THREAD

**Issue by aburgessPARS** [dfinke/ImportExcel#513]
When copying a worksheet to another workbook, the images inside the worksheet will change to images that are currently in the destination workbook. I have encountered this with different images and different worksheets. Attached is a simple example that demonstrates this issue with 2 workbooks and the sample powershell script demonstrating the issue. The powershell script has been renamed from .ps1 to .txt so I could upload it. Please let me know if there is anything I can do to help. I have tried multiple workarounds.
[IssueTest.txt](https://github.com/dfinke/ImportExcel/files/2713128/IssueTest.txt)
[StatusAllRed.xlsx](https://github.com/dfinke/ImportExcel/files/2713123/StatusAllRed.xlsx)
[ColorStatus.xlsx](https://github.com/dfinke/ImportExcel/files/2713124/ColorStatus.xlsx)

**Comment by jhoneill** [comment]
Hi. I can reproduce this (and thanks for the samples - it makes it so much easier when you can see the problem easily.) 
 I think we're hitting a bug in the EPPlus library, I'll check against a newer version. It's a strange one, because it doesn't get every image wrong.    I tried making the picture names unique, but that didn't change anything. I'll look further, but I suspect I'm going to be filing a bug report against EPPlus.

**Comment by aburgessPARS** [comment]
First of all. Thank you for checking so quickly! I have been doing a lot of digging in the last day. not looking at the code, but looking at the outputs of my different tests.  The problem only happens when there are images in the source worksheet and destination workbook. This only happens for images of the same type. I have learned more about excel in the last day than I ever wanted :-).  What is happening is that the image in the sheet being copied is not being copied into the xl/media folder if the same image name is already there.

What needs to happen for images that already exist in the media folder is for the new images to be renamed from something like image1.jpg to imageX.jpg where X is the next sequential number after all of the existing images.

In the xl\drawings\_rels\drawingX.xml.rels (where x is the new spreadsheet copied in) change the target for all images being renamed from the old image name to the new images name. Example (change image1 to imageX).
	<Relationship Id="rId1" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/image" Target="../media/image1.jpg"/>

Lastly, add new overrides for each newly added image to [Content_Types].xml
	<Override ContentType="image/jpeg" PartName="/xl/media/imageX.jpg" />

I was able to successfully correct by manually by unzipping, making the changes mentioned above and rezipping, and then opening it up in excel. 

I hope this helps.
